.env

```diff
@@ -0,0 +1,6 @@
+# Documentation:
+# https://www.gatsbyjs.org/docs/environment-variables/
+# https://docs.netlify.com/configure-builds/environment-variables/#declare-variables
```

Comment:
If we want to do something like store secrets/keys we could use Netlify functions. I set up a basic example and linked to documentation here.